src/App.tsx

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const items = getNumbers(1, 42);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<string>('5');
```

Comment:
The state 'perPage' is initialized with a string value, but it's used as a number later in the code. This could lead to confusion and potential bugs. Consider initializing it as a number to ensure type consistency.